src/app/visual/action-button/action-button-visual.component.html

```diff
@@ -1,17 +1,34 @@
-<div id="screenshot-action-button">
-    <sky-action-button (actionClick)="buttonClicked()">
-      <sky-action-button-icon
-        iconType="filter"
-      ></sky-action-button-icon>
-      <sky-action-button-header>
-        Build a new list
-      </sky-action-button-header>
-      <sky-action-button-details>
-        Start from scratch and fine-tune with filters
-      </sky-action-button-details>
-    </sky-action-button>
+<div
+  id="screenshot-action-button"
+  style="display:inline-block;"
+  [ngStyle]="{
+    'padding.px': containerPadding
```

Comment:
This was done so that the original screenshots weren't affected (and padding only added for the `focus` screenshot).